### ISSUE TITLE
Accord.Net Simplest face detection is too stable

### ISSUE THREAD

**Issue by blaisexen** [accord-net/framework#456]
This is my Simplest Multiple Face Detection,
**I'm so surprised that the rectangle is very stable if you don't move your face.**

Because in Emgu.Net even if you make your face a statue or cemented the rectangle is not stable,
But stays side in the face.

Tested that Accord.Net face detection is very much cooler than OpenCv face detection.

BUT,
if you move your face even 1 centimeter, the rectangle moves and flashes fast,
and doesn't stay at your face, it moves right to left to right to up to down to up to left.

here's the simplest code, I hope someone can correct it.
Please try, I'm listening from your solution, for a test, and I'm gonna use it IF it's better.
```
        // New frame received by the player
        private void videoSourcePlayer_NewFrame(object sender, ref Bitmap image)
        {
            UnmanagedImage im = UnmanagedImage.FromManagedImage(image);
            float xscale = image.Width / 320;
            float yscale = image.Height / 200;
            ResizeNearestNeighbor resize = new ResizeNearestNeighbor(320, 200);
            UnmanagedImage downsample = resize.Apply(im);
            Rectangle[] regions = detector.ProcessFrame(downsample);

                if (regions.Length > 0)
                {
                    for (var i = 0; i < regions.Length; i++)
                    {

			//Resize Rectangle here?
                        regions[i].X += (int)(regions[i].Height * 0.08);
                        regions[i].Y += (int)(regions[i].Width * 0.16);
                        regions[i].Height -= (int)((regions[i].Height * 0.16));
                        regions[i].Width -= (int)(regions[i].Width * 0.08);                        
			
			//Assign faces to rectangles
                        Rectangle face = regions[i];
                        Rectangle window = new Rectangle((int)((face.X + face.Width / 2f) * xscale), (int)((face.Y + face.Height / 2f) * yscale), 1, 1);
                        window.Inflate((int)(0.5f * face.Width * xscale), (int)(0.5f * face.Height * yscale));

                        //Draw Rectangle in the face
                        marker = new RectanglesMarker(window);
                        marker.MarkerColor = Color.DeepSkyBlue;
                        marker.ApplyInPlace(im);
                        image = im.ToManagedImage();
                    }
                }
        }
```
thank you

**Comment by blaisexen** [comment]
I hope someone understand what I meant.

**Comment by blaisexen** [comment]
unclear

**Comment by cesarsouza** [comment]
Hi @blaisexen,

I am really sorry that I couldn't give the proper attention to this issue at the time, back in February.

Do you think you could send me an example of a video that you were trying to feed the face detection example from Accord.NET and what you were expecting to see?

I think I might have figured out what you meant by being surprised that the rectangle stayed so stable. I am about to commit a new example showing how to do multi-face detection and I think it should show how to overcome the problem you had found in the beginning of this year.

I know that it might be already late for you, since I guess you had moved on with other projects since that time. But I hope that the new example can help other users of the framework who are just starting trying to make the face detection example work.

Regards,
Cesar

**Comment by blaisexen** [comment]
@cesarsouza 
[This is the last compilation I had](http://www.mediafire.com/file/acyn37hihqodm3q/Accord.Face_Detection.zip)

I'm sorry if had not changed anything, it's the simplest face detection from Accord.Net I mix-up.

issues, 
face detection movements are not stable and toooooo slow.

**Comment by blaisexen** [comment]
@cesarsouza 
NOW....

I've found out issues!
Changing from
detector.MinSize = new Size(30, 30) //LESSER THAN EQUAL TO 31 SLOWS THE DETECTION;
TO:
detector.MinSize = new Size(32, 32);

AND the
HaarObjectDetector **ScaleFactor** from 1.1f OR 1.5f 
TO
2.5f

IMPROVES the face detection.


only issue remains.......
is the **movement** of the face rectangle....
it is stable IF you steady your face....

BUT, if you move your face a little tiny.... 
IT will randomized rectangle movement from left or right or top or bottom.

That's ALSO the issue I've found out FROM the original AFORGE....

I wish I have more knowledge but I could not do anything...
Sorry!
